### DISCUSSION TITLE
how to solve the problem of "OpenICLInfer Fail"?

### DISCUSSION THREAD

**Discussion by SijunWang** [open-compass/opencompass#556] [Q&A]
OpenCompass - WARNING - task OpenICLInfer[llama-2-7b-hf/lukaemon_mmlu_professional_law_0] fail, see

OpenCompass - WARNING - task OpenICLEval[llama-2-7b-hf/lukaemon_mmlu_college_computer_science] fail, see

**Comment by tonysy** [comment]
Please check the log in *.out file
